graph/client/src/main.tsx

```diff
@@ -9,30 +9,58 @@ import { inspect } from '@xstate/inspect';
 import { App } from './app/app';
 import { ExternalApiImpl } from './app/external-api-impl';
 import { render } from 'preact';
+import { ErrorToastUI, ExpandedTargetsProvider } from '@nx/graph/shared';
+import { ProjectDetails } from '@nx/graph-internal/ui-project-details';
+import { ErrorPage } from './app/ui-components/error-page';
 
-if (window.useXstateInspect === true) {
-  inspect({
-    url: 'https://stately.ai/viz?inspect',
-    iframe: false, // open in new window
-  });
-}
-
-window.externalApi = new ExternalApiImpl();
-const container = document.getElementById('app');
+if (window.__NX_RENDER_GRAPH__ === false) {
```

Comment:
Instead of adding to the `main.tsx` file, would it be better to create a separate entry for Console? If we add it to `webpack.config.js`.


```js

module.exports = {
  ...
  plugins: [
    new NxAppWebpackPlugin({
      ...
      additionalEntryPoints: [
        {
          entryName: 'main.console',
          entryPath: 'graph/client/src/main.console.tsx',
        },
      ],
    }),
    ...
  ],
};
```

Note: I found a bug with entryPath not supporting relative paths. In the meantime the above will work. And in Console, you can check if `main.console.js` exists or not.

Reply:
Yes, this was the very first approach I had. What I don't like about this is that it bundles everything in that seperate entry file, so the graph bundle will go from 3mb -> 5mb. 
It's not the end of the world but it didn't feel great so I went with the current solution instead. Lmk what you think about that :)